### ISSUE TITLE
[Android] Clocks and Clocktower spin really fast

### ISSUE THREAD

**Issue by fireflamesniper** [Waterdish/2ship2harkinian-Android#1]
- [ *] I have checked the known issues and searched Github to ensure this issue has not already been reported.
- [*] This is not a feature request, I know those belong on Discord/Github Discussions.

### Can you reproduce? If so please list the steps:
- [ ] No, I cannot reproduce this bug
- [* ] Yes, here are the steps to reproduce the bug:

### System Information:
- OS: Android 14
- GFX Backend: OpenGL
- Game Version: 1.0.1
- Rom Version: US 1.0

**Comment by fireflamesniper** [comment]
Not exactly sure why my descriptions or links aren't showing up in the big report... But here is the video of this happening 

https://cdn.discordapp.com/attachments/1172523005558145154/1249229948242231336/Screen_Recording_20240609_011152_2_Ship_2_Harkinian.mp4?ex=66692e8d&is=6667dd0d&hm=b5632daa016b867b1b5b1c176882d3b5bc29545c1f60413b43c74151bae40d19&

https://cdn.discordapp.com/attachments/1245431249707991200/1249225378581708811/Screen_Recording_20240609_005443_2_Ship_2_Harkinian.mp4?ex=66692a4b&is=6667d8cb&hm=c230206e8abbced42cb40fd54af9f11907d2e40202e733e3503517ee697019ef&

Basically the clock on walls and the clock tower itself will have its rings spinning really fast until it hits the top of the minute, once the second hand passes the bottom, it starts spinning really fast again

**Comment by NotAn127** [comment]
Its not only you. It is happening to everyone, and it may be due to the fps as to why it spins fast

**Comment by svinsmoke212** [comment]
> Its not only you. It is happening to everyone, and it may be due to the fps as to why it spins fast

This issue happens to the Mac users. I think when HM64 releases a new version this would probably be fixed.
